src/wallet/sqlite.cpp

```diff
@@ -61,6 +61,42 @@ SQLiteDatabase::SQLiteDatabase(const fs::path& dir_path, const fs::path& file_pa
     }
 }
 
+void SQLiteBatch::SetupSQLStatements()
+{
+    std::string read_sql = "SELECT value FROM main WHERE key = ?";
+    std::string insert_sql = "INSERT INTO main VALUES(?, ?)";
+    std::string overwrite_sql = "INSERT OR REPLACE INTO main VALUES(?, ?)";
+    std::string delete_sql = "DELETE FROM main WHERE key = ?";
+    std::string cursor_sql = "SELECT key, value FROM main";
```

Comment:
83952c59940e4b03bf63b48c0f83b460f988d4be
Why not declare all strings as `const char* const` and drop all of the following `c_str` conversions?

Reply:
Inlined all of these.